projects/gsc/experiments/base.py

```diff
@@ -177,11 +181,53 @@
     batch_sizes=[4, 16],  # 4 for the first epoch and 16 for the remaining
     val_batch_size=1000,
     batches_in_epoch=sys.maxsize,
+)
```

Comment:
Just a side note: `DEFAULT_BASE` should replicate the 97% results. I'm not sure about `DEFAULT_SPARSE_CNN`. In our discussion, we made sure `DEFAULT_BASE` replicated the best result from the `whydense` project. Could be helpful to make sure this is clear in this experiment file if not already.